solutions/base-layer-account-settings/main.tf

```diff
@@ -0,0 +1,23 @@
+########################################################################################################################
+# Base Account
+########################################################################################################################
+
+module "basic_account" {
+  source = "../.."
+  providers = {
+    logdna.at = logdna.at
+    logdna.ld = logdna.ld
+  }
+  region                      = var.region
+  resource_group_name         = "${var.prefix}-resource-group"
+  cos_instance_name           = "${var.prefix}-cos-instance"
+  cos_bucket_name             = "${var.prefix}-cos-bucket"
+  trusted_profile_name        = "${var.prefix}-trusted-profile"
+  cos_resource_key_name       = "${var.prefix}-cos-key"
+  cos_target_name             = "${var.prefix}-target"
+  activity_tracker_locations  = [var.region, "global"]
```

Comment:
Added default to root level, assuming this is something that doesn't need a variable (because we always want logs from all regions) removing this line that provides the list.

Reply:
I think its fine to not expose it in the DA unless someone explicitly asks for it